internal/cmd/tracking-issue/main.go

```
@@ -153,8 +162,13 @@ func (ws Workloads) Markdown() string {
 	sort.Strings(assignees)
 
 	var b strings.Builder
+
+	for _, label := range tracking.LabelWhitelist {
+		b.WriteString(fmt.Sprintf(labelMarkerFmt, label))
+	}
```

What makes it necessary to write these whitelisted label comment markers automatically? I thought this was something one would right manually in the issue body.

---

since we write the issue markdown in one pass from scratch each time (which I think is ~declarative & good), we also need to rewrite the label markers or they'll disappear on the next update. i didn't want to enforce that the label markers have to be outside the `WORK` section because that would confuse people when labels inside the `WORK` section disappear for no reason. 

now that I think about it, this makes labels outside the work section persist without having an effect which is also confusing, so i think the easiest change is to parse all lines of the issue body for label markers, and then persist them inside the `WORK` section.

---

I think the code would be much easier to understand to me if the `LABEL` annotations would be outside the `WORK` section, and for the relation to be explicitly documented. Would you be OK with making this change?

---

yup!

---

hrm, if we want to avoid the confusing behaviour that LABELs inside the WORK section are silently discarded, this will make the code more convoluted and confusing. but i think it's OK to just have that property, because i agree with you that intuitively the labels should go outside the WORK section.